symphony/content/content.publish.php

```diff
@@ -495,7 +495,10 @@ public function __viewIndex()
 
         // Flag filtering
         if (isset($_REQUEST['filter'])) {
-            $filter_stats = new XMLElement('p', '<span>– ' . __('filtered (%d of %d)', array($entries['total-entries'], EntryManager::fetchCount($section_id))) . '</span>', array('class' => 'inactive'));
+            $filter_stats = new XMLElement('p', '<span>– ' . __('%d of %d entries (filtered)', array($entries['total-entries'], EntryManager::fetchCount($section_id))) . '</span>', array('class' => 'inactive'));
+            $this->Breadcrumbs->appendChild($filter_stats);
```

Comment:
Haha, you just voted for always having a count and now you are already planning the pull-out :grinning:
I don't mind, I just noticed …